persistence/trees/trees.go

```diff
@@ -101,6 +101,16 @@ func (t *treeStore) GetTree(name string) ([]byte, kvstore.KVStore) {
 	return nil, nil
 }
 
+// GetTreeHashes returns a map of tree names to their root hashes for all
+// the trees tracked by the treestore, excluding the root tree
+func (t *treeStore) GetTreeHashes() map[string]string {
```

Comment:
Can you add a unit test for this?

Reply:
Not without possible conflicts in #861 - it would require the same boilerplate setup functions to be copied over into `trees/trees_test.go` I can do this if you think its 100% necissary but this logic is pretty straighforward, I can leave a skip test that can be implemented after #861 is in?